packages/next-app/src/components/PoolMetadataFormModal.tsx

```diff
@@ -28,10 +44,34 @@ const inputTypes = [
 
 export function PoolMetadataFormModal({
   children: trigger,
+  context,
+  data,
 }: {
   children: React.ReactNode;
+  context: "add" | "edit";
```

Comment:
minor thing but `context` might be confusing here. maybe `mode` or `type` is best suited to describe it?

Reply:
agreed. I think `mode` is a good option. thanks sir